packages/fast-tooling/src/web-components/css-box-model/css-box-model.template.ts

```diff
@@ -10,173 +25,256 @@ export const cssBoxModelTemplate = (context: ElementDefinitionContext) => html<
     CSSBoxModel
 >`
     <template>
-        <div class="grid grid-margin">
-            <div class="item item-label">margin</div>
-            <div class="item item-top">
+        <div class="section">
+            <label for="margin" class="section_label">Margin</label>
```

Comment:
just a note that the `for` and `id` may not be unique on a page, this is fine for now since it's alpha, but we should write an issue to track how we can allow the user to specifically set these strings to avoid conflict